core/embed/trezorhal/optiga/optiga_commands.c

```diff
@@ -185,18 +210,44 @@ optiga_result optiga_serialize_metadata(const optiga_metadata *metadata,
   return OPTIGA_SUCCESS;
 }
 
+bool optiga_compare_metadata(const optiga_metadata *expected,
```

Comment:
I just want to emphasize that this function evaluates the metadata as the same even if  `stored` contains an extra tag.

Reply:
At least, I would mention it in a comment.

Reply:
e869b104b646d4edbb93c773f4ffd02e653164b9